src/xml2js.coffee

```diff
@@ -92,6 +106,7 @@ class exports.Builder
   buildObject: (rootObj) ->
     attrkey = @options.attrkey
     charkey = @options.charkey
+    cdata = @options.cdata
```

Comment:
Ah, alright, my bad. Actually I think changing `->` to `=>` is the option that I'd prefer.